D2211R0.md

```diff
@@ -227,9 +227,9 @@ struct FlagsV1 {
 
 ```C++
 inspect(flagsV1) {
-  [false, false] => /*...*/,
-  [true , false] => /*...*/,
-  [_    , true ] => /*...*/,
+  [false, false] => /*...*/;
+  [true , false] => /*...*/;
+  [_    , true ] => /*...*/;
```

Comment:
Should this and some patterns below be `__` for consistency with the rest of paper?